app/models/manageiq/providers/openstack/cloud_manager/flavor.rb

```diff
@@ -1,6 +1,11 @@
 class ManageIQ::Providers::Openstack::CloudManager::Flavor < ::Flavor
   include ManageIQ::Providers::Openstack::HelperMethods
 
+  supports :create
+  supports :delete do
+    unsupported_reason_add(:delete, _("The Flavor is not connected to an active Provider")) if ext_management_system.nil?
```

Comment:
shouldn't it be checking whether `ext_management_system.nil?` on `:create` as well?

Reply:
> shouldn't it be checking whether ext_management_system.nil? on :create as well

There is no flavor instance to check on create since we're trying to create it.  The supports feature blocks are only invoked on the instance level not the class level.

>  in fact, the original code in this file seems to have been checking only on :create and not on :delete

Yeah I don't think those validations make any sense (see https://github.com/ManageIQ/manageiq/pull/20332#issuecomment-654343352).  We're validating that when we create a flavor that it has an EMS but it isn't created yet.

Reply:
Going to simplify this for this bugfix and leave the validates_* methods and just move the `supports *` calls